eth/db/storage.py

```diff
@@ -413,6 +417,9 @@ def _validate_flushed(self) -> None:
                 f"StorageDB had a dirty journal when it needed to be clean: {journal_diff!r}"
             )
 
+    def get_accessed_slots(self) -> Tuple[int, ...]:
```

Comment:
Why not have this return a set?

Reply:
Just a general aversion to returning mutable data structures.